base_comment_template/models/comment_template.py

```diff
@@ -42,3 +42,10 @@ def _compute_comment_template_ids(self):
                 domain = safe_eval(template.domain)
                 if not domain or record.filtered_domain(domain):
                     record.comment_template_ids = [(4, template.id)]
+
+    def render_comment(self, comment):
+        self.ensure_one()
+        comment_texts = self.env["mail.template"]._render_template(
```

Comment:
I've the feeling that could be proper to use or inherit directly from `mail.render.mixin` abstract model instead of using `mail.template` maybe I missed something ?